StripeUICore/StripeUICore/Source/Elements/TextField/TextFieldElement.swift

```diff
@@ -109,7 +114,12 @@ import UIKit
 
     /// Call this to manually set the text of the text field.
     public func setText(_ text: String) {
-        self.text = sanitize(text: text)
+        if configuration.isEditable {
+            self.text = sanitize(text: text)
+        }
+        else {
+            self.text = text
+        }
```

Comment:
Having this dupe'd code in two places doesn't feel right, and not sanitizing due to not editing doesn't make sense to me.

What happens if we remove this? Maybe we should have a text field configuration that would handle the sanitization correctly, similar to what we do for last 4 configuration.